docs/site/Creating-components.md

```diff
@@ -109,12 +109,13 @@ In some cases, the Provider may depend on other parts of LoopBack; for example t
 
 ```ts
 import {Provider} from '@loopback/context';
+import {ServerRequest} from 'http';
 const uuid = require('uuid/v4');
 
 class CorrelationIdProvider implements Provider<string>{
-  constructor(@inject('http.request') private request) {}
+  constructor(@inject('rest.http.request') private request: ServerRequest) {}
```

Comment:
Have you considering using `RestBindings.Http.Request` instead of the hard-coded string? See https://github.com/strongloop/loopback-next/blob/6c94752ae5244d04311d8a7e3aacfffee2ac9f3d/packages/rest/src/keys.ts#L33